src/server.js

```diff
@@ -90,12 +110,15 @@ const main = async () => {
    * @apiParam {String} properties.appId The application
```

Comment:
We should update the docs for this endpoint to include the new way of calling it, and explicitly list the `distinct_id` property here.